Detections/SecurityEvent/gte_6_FailedLogons_10m.yaml

```diff
@@ -18,11 +18,14 @@ relevantTechniques:
 query: |
 
   let timeframe = 10m;
+  let threshold = 20;
   SecurityEvent 
-  | where TimeGenerated > ago(2*timeframe) 
+  | where TimeGenerated >= ago(timeframe)
   | where EventID == "4625"
```

Comment:
In other queries you have dropped the " around the EventID number

Reply:
Yes, thanks.